buildinggame/src/main/java/com/gmail/stefvanschiedev/buildinggame/Main.java

```diff
@@ -426,4 +432,23 @@ public void loadPlugin(boolean reload) {
     public static Main getInstance() {
 		return instance;
 	}
-}
\ No newline at end of file
+
+	/**
+	 * Returns false if the user using old java version, that is not supports.
```

Comment:
This should be changed to something like "Returns false if the user is using an old Java version, that is not supported." to fix the grammar issues.